We need to actually flush the async File to ensure everything gets written.

oli-obk opened the pull request:
The `Drop` impl won't do it for us like the `Drop` impl of `std::fs::File` does

fixes #46

DerKarlos commented:
No file errors any more. No glb - log outputs either, I added it again.
But:  (the cache is filled)
After about a minute, anything gets slow,  3-4 ms/frame
I am still testing

oli-obk commented:
> log outputs either, I added it again.

it's so spammy, which is why I removed it. Can we move it to `debug` instead? so it isn't shown by default, but can still be enabled?

oli-obk commented:
> After about a minute, anything gets slow, 3-4 ms/frame

maybe we're loading/unloading tiles at the sky boundary all the time?

DerKarlos commented:
debug! is fine (It's a run option, isn't it?)
So we need more debug for un/loading :-)
I try to do it ...

DerKarlos commented:
With Web it's about the same: gets really slow. Tile loading seems to stop but takes just a minute
